packages/esm-patient-biometrics-app/src/biometrics/biometrics.resource.tsx

```diff
@@ -1,6 +1,7 @@
 import useSWR from 'swr';
 import { fhirBaseUrl, FHIRResource, useConfig, openmrsFetch } from '@openmrs/esm-framework';
 import { calculateBMI } from './biometrics-helpers';
+import { toDateWithoutSeconds } from '@openmrs/esm-patient-common-lib';
```

Comment:
is this leveraging the recent PR from brandon?

Reply:
It's not. I didn't write a function that removes seconds from a date.

Why were multiple lines being generated in the first place? i.e. why does some data seem to have multiple copies that differ by seconds?

Reply:
The reason why some data has multiple copies is because it is being capture at different times, the difference is in milliseconds.

Reply:
Oh okay, so the data is all submitted to the server at the same moment, and the time difference is between the server saving each thing to the database. That is very goofy.

The way you're solving this is okay. A more formally correct way to do it would be to check whether the difference between times is less than some value. Then it behaves correctly even in the unlikely scenario where the millisecond difference runs over a minute boundary. It would also be more expressive of the intent of the code (though it wasn't hard to understand what you did).

Reply:
@brandones i have removed the date manipulation function to be done correctly on a different PR

Reply:
Ok @donaldkibet can you update the PR title to reflect what this PR now does (removes biometrics from vitals display, as best I can tell)?

Reply:
@brandones is there further feedback on this?

Reply:
Does it fix the wrong BMI calculations? Looking at the code I don't understand how it does that.